packaging/hudi-hadoop-mr-bundle/pom.xml

```diff
@@ -143,17 +143,7 @@
     <dependency>
       <groupId>org.apache.avro</groupId>
       <artifactId>avro</artifactId>
-      <scope>${mr.bundle.avro.scope}</scope>
+      <scope>compile</scope>
     </dependency>
   </dependencies>
-
-  <profiles>
-    <profile>
-      <id>mr-bundle-shade-avro</id>
```

Comment:
@bvaradar @n3nash : call this to your attention. is removing the profile going to affect the custom payload implementations at Uber?

Reply:
@umehrot2 (cc @vinothchandar ) I will get back on this by today EOD.

Reply:
(cc @n3nash ) Yeah, this would mean that we need to employ the same package relocation in the jar carrying custom record payloads. As discussed in the earlier threads, there is no way around it. @umehrot2 : We would need to document this caveat in Release Notes and add documentation on how to shade it. Can you create a ticket to track  this ?

Reply:
@vinothchandar @bvaradar Yes, this will affect the custom payload implementation on the reader side. But we are anyways going to make some changes in how the payload packages are loaded so we should be able to absorb this change as part of those considerations.